### DISCUSSION TITLE
Melody on the Arch Linux User Repository

### DISCUSSION THREAD

**Discussion by ilai-deutel** [yoav-lavi/melody#44] [General]
Hi there,

First of all, great project! I created a [`melody` package on the Arch Linux User Repository]( https://aur.archlinux.org/packages/melody). This makes it easier for Arch Linux users to install melody on their systems.

Let me know if you have any comments, I'm also happy to can create a PR to add installation instructions to the README.

**Comment by yoav-lavi** [comment]
Thank you @ilai-deutel! That's great!
Would be happy to merge a PR for that

**Reply by jyooru** [reply]
I'm trying to package Melody for nixpkgs (nixos/nixpkgs#160985), and was wondering if you have a preference for what the package should be named - right now I have it packaged under `melody-cli`, should I move it to `melody`?

I named it `melody-cli` as that was what the crate providing the CLI was called.

**Reply by yoav-lavi** [reply]
@jyooru to be honest it was originally `melody_cli` due to a naming conflict (`melody` was taken), the differentiation makes sense for crates since the compiler is also exposed but for non Rust packages `melody` might be best

**Reply by yoav-lavi** [reply]
@jyooru thanks for the contribution!

**Comment by yoav-lavi** [comment]
@ilai-deutel @jyooru Added [this](https://github.com/yoav-lavi/melody/blob/main/README.md#community) section in the README.md in the meantime, please update with install instructions in a PR if you can, thank you!
